packages/components/src/styles/components/form/radio-card.scss

```diff
@@ -64,21 +64,32 @@
 
   &--checked,
   &.mock-checked {
-    border-color: var(--token-color-focus-action-internal);
+    border-color: var(--token-color-palette-blue-300);
 
     .hds-form-radio-card__control-wrapper {
       background-color: var(--token-color-surface-action);
       border-color: var(--token-color-border-action);
     }
+
+    &:hover,
+    &.mock-hover {
+      border-color: var(--token-color-palette-blue-400);
+    }
   }
 
   &--disabled,
   &.mock-disabled {
     background-color: var(--token-color-surface-interactive-disabled);
     border-color: var(--token-color-border-primary);
     box-shadow: none;
+    color: var(--token-form-control-disabled-foreground-color);
```

Comment:
Set the base color for child elements within the `RadioCard` in it's disabled state.